processes/decision_process.rst

```diff
@@ -128,9 +127,9 @@ At least one member of the DM TCT_ will read each RFC to determine if it might a
 Responsibility and delegation
 -----------------------------
 
-For project management purposes, RFCs are formally proposals made to the Project Manager, Project Scientist and Project Engineer who by default are responsible for everything in DM (they "own" all problems).
-As owners, they have the final word in accepting or rejecting all proposals.
-Functionally, they delegate that ownership---the right and responsibility to make decisions---to others within the team (e.g. the System Architect, Interface Scientist, group leads, etc.) who are expected to delegate it even further.
+For project management purposes, RFCs are formally proposals made to the Subsystem Lead who by default is responsible for everything in DM (the Subsystem Lead "owns" all problems).
```

Comment:
This feels wrong as it skips many levels of the hierarchy. But I don't have anything better to replace it with.